src/ui/pages/SidePage/SidePage.test.tsx

```diff
@@ -68,15 +74,6 @@ describe("Side Page: wallet connect", () => {
     await waitFor(() => {
       expect(getByTestId("alert-decline-connect-confirm-button")).toBeVisible();
     });
-
-    act(() => {
-      fireEvent.click(getByTestId("alert-decline-connect-confirm-button"));
-      fireEvent.transitionEnd(getByTestId("side-slider"));
-    });
-
-    await waitFor(() => {
-      expect(dispatchMock).toBeCalledWith(setPauseQueueIncomingRequest(false));
-    });
```

Comment:
Why are these removed?

Reply:
Because I can't trigger click event of button on modal => I mocked `IonModal` and these code is unnecessary now.